### ISSUE TITLE
RPM is missing from supported package managers

### ISSUE THREAD

**Issue by brianwcook** [containerbuildsystem/cachi2#579]
$title : please add RPM to 

https://github.com/containerbuildsystem/cachi2?tab=readme-ov-file#package-managers

**Comment by brianwcook** [comment]
well I see dnf is there, but is it fair to ask to move it from 'planned' to 'implemented' now?

**Comment by slimreaper35** [comment]
Closing (added in #1024)

**Comment by eskultety** [comment]
> Closing (added in [#1024](https://github.com/hermetoproject/hermeto/pull/1024))

Not true, this issue is about a different backend that is to supersede and eventually deprecate the raw RPM backend in its current shape -> reopening!

**Comment by eskultety** [comment]
A tiny update on this - we're still waiting for the DNF manifest support to land in upstream, including python API bindings which we could consume, so the dependent work is in progress, there's just no ETA on this at the moment so we're blocked here for the time being.
